### ISSUE TITLE
M138 Absolute VCC

### ISSUE THREAD

**Issue by jacky4566** [sparkfunX/Satellite_Transceiver_Breakout__Swarm_M138#4]
What is the absolute VCC for the M138? 
The datasheet says 5.0V and SparkFun schematic shows this connected directly to USB which is allowed to deviate up to 5.5V

Could be a problem?

**Comment by PaulZC** [comment]
Hello Jackson (@jacky4566 ),

The latest M138 datasheet quotes an input voltage of 3.0V to 5.0V. It also quotes absolute limits of 3.0V to 5.0V, which is unusual.

However, an earlier version of the datasheet - the one we were working from when we designed the board - quoted an input voltage range and absolute limits of 3.0V to 5.5V.

I believe the USB voltage requirement is 5.0V +/- 5%. I.e. 5.25V maximum? I believe the +0.5V deviation only applies during transitions (i.e. at switch-on)?

In summary, I believe we're OK, but I will check.

Best wishes,
Paul

**Comment by PaulZC** [comment]
Hi Jackson,

We've been in touch with Swarm and they've confirmed that the modem can withstand 5.5V, as an _absolute_ maximum (non-continuous). Regular operation should be between 3.0V and 5.0V.

Best wishes,
Paul
